src/mbgl/renderer/render_source.cpp

```diff
@@ -71,4 +71,15 @@ bool RenderSource::isEnabled() const {
     return enabled;
 }
 
+std::vector<UnwrappedTileID> RenderSource::getRenderTileIDs() {
```

Comment:
Can this be cached? Right now, '_clippable_' render tiles are filtered on every frame for all layers that require clipping.

Reply:
Removed this in favor of using `renderTiles` of the current layer. I initially tried to do this but observed some mismatches. I think https://github.com/mapbox/mapbox-gl-native/pull/14385 fixed the underlying issue.